File: eRecipes/eRecipes.Service/Database/Obavijest.cs
using System;
using System.Collections.Generic;

namespace eRecipes.Service.Database;

public partial class Obavijest
{
    public int ObavijestId { get; set; }
    public string Naslov { get; set; } = null!;

    public string Sadrzaj { get; set; } = null!;

    public int KorisnikId { get; set; }

    public DateTime DatumSlanja { get; set; } = DateTime.Now;

    public virtual Korisnik Korisnik { get; set; } = null!;
}

File: eRecipes/UI/erecipes_desktop/lib/models/obavijest.dart
import 'package:erecipes_desktop/models/korisnik.dart';
import 'package:json_annotation/json_annotation.dart';

part 'obavijest.g.dart';

@JsonSerializable()
class Obavijest {
  int? obavijestId;
  String? naslov; 
  String? sadrzaj;
  int? korisnikId;
  Korisnik? korisnik;
  DateTime? datumSlanja;
  bool? procitano;
  
     Obavijest({
    this.obavijestId,
    this.naslov,
    this.sadrzaj,
    this.korisnikId,
    this.korisnik,
    this.datumSlanja,
    this.procitano,
  });
  
  factory Obavijest.FromJson(Map<String,dynamic> json)=> _$ObavijestFromJson(json);

  Map<String,dynamic> toJson() => _$ObavijestToJson(this);
}